api/types/swarm/common.go

```diff
@@ -17,7 +17,7 @@ type Meta struct {
 // Annotations represents how to describe an object.
 type Annotations struct {
 	Name   string            `json:",omitempty"`
-	Labels map[string]string `json:",omitempty"`
+	Labels map[string]string `json:"Labels"`
```

Comment:
Thank you for the review. I removed the _omitempty_ because my tests show 'null' as output with _omitempty_ left as is, while including all other changes of the PR. (I think the empty labels map is lost in marshalling/unmarshaling due to _omitempty_. The test steps are listed in the PR description). I did not find any client-side code that makes changes to the retrieved object, so avoided adding such code (e.g. in client/service_inspect.go). Is fixing the nil labels map on the client-side the preferred approach? (in that case, I can update the PR accordingly)